### PR TITLE
Mark master gh7

### PR DESCRIPTION
**Pull request by mark-lunarg**
Fixup invalid C++ variable names in the codebase.

### CODE REVIEW

File: layers/core_validation.h [hunk 1]
```diff
@@ -292,12 +292,12 @@ typedef struct _DESCRIPTOR_POOL_NODE {
...
-    ~_DESCRIPTOR_POOL_NODE() {
+    ~DESCRIPTOR_POOL_NODE() {
         delete[] createInfo.pPoolSizes;
         // TODO : pSets are currently freed in deletePools function which uses freeShadowUpdateTree function
         //  need to migrate that struct to smart ptrs for auto-cleanup
     }
-} DESCRIPTOR_POOL_NODE;
+};
 
 typedef struct stencil_data {
```
Comment (tobine) [inline layers/core_validation.h]: did you intentionally skip this guy?

**Comment by tobine** [comment]
LGTM :+1:

File: layers/core_validation.h (hunk 1 above)
Comment (mark-lunarg) [inline layers/core_validation.h]: Yeah -- this only addressed the invalid variables problem and while I was there I updated the modified lines to use the new coding guidelines.  Fixing all of them will be a different issue, and man it was hard to leave them as-is.

**Comment by mark-lunarg** [comment]
Validated and pushed to master.

**Pull request merged**